plugins/generator-1.19.2/forge-1.19.2/mappings/projectiles.yaml

```diff
@@ -1,40 +1,53 @@
 _default:
   - Snowball
   - EntityType.SNOWBALL
+  - Items.AIR

```

Comment:
Shouldn't most entries here actually map to an item? Snowballs, eggs and others do have their item representation, except for 3 bottom-most projectiles.

Reply:
Yes, they have. However, the current code of projectiles is only made for custom arrows, while snowballs/eggs and others like fireballs are other types of projectiles (throwable, fireballs, etc.). I already plan to add support for them, but I'm just waiting for this PR to be merged as it would make the PR way too big and it would combine too many things.